### ISSUE TITLE
404 GET /nbextensions/nbtutor/js/nbtutor.deps.min.js.js

### ISSUE THREAD

**Issue by panzerballett** [lgpage/nbtutor#49]
I encountered a strange behavior when trying to install `nbtutor`.

The strange thing is that I was able to install it in my `~/tmp` directory and everything worked fine. When I was trying to install it in the project directory I wanted it, I got the following error message:

```
[W 06:51:57.701 NotebookApp] 404 GET /nbextensions/nbtutor/js/nbtutor.deps.min.js.js (127.0.0.1) 1.260000ms referer=http://localhost:8888/notebooks/src/jupyter/Untitled.ipynb?kernel_name=python3
```

This is in ubuntu 20.04; python 3.8.10.

The install process (in both directories) was:

```
$ python3 -m venv nbtutor_venv 
$ source nbtutor_venv/bin/activate
(nbtutor_venv) $ pip install --upgrade pip
(nbtutor_venv) $ pip install nbtutor
(nbtutor_venv) $ jupyter nbextension install --overwrite --py nbtutor --sys-prefix
(nbtutor_venv) $ jupyter nbextension enable nbtutor --py --sys-prefix
(nbtutor_venv) $ jupyter notebook
```

I do not see any files in the project directory that might affect the behavior or jupyter... Any ideas?

Thanks a lot for such a great tool & best regards!

**Comment by vincentxavier** [comment]
`nbtutor.min.js` and `nbtutor.min.css` are missing from the repo, so they are not included in the distribution.
See https://github.com/vincentxavier/nbtutor/commit/c1da217c44a84bd13b190c72eb0ef1ba66bbc90a

**Comment by lgpage** [comment]
This should be fixed in #62.
Please reopen this ticket if the issue persists
